src/main/java/com/gtnewhorizon/structurelib/Tags.java

```diff
@@ -0,0 +1,12 @@
+package com.gtnewhorizon.structurelib;
+
+// Use this class for Strings only. Do not import any classes here. It will lead to issues with Mixins if in use!
+
+public class Tags {
```

Comment:
This should be referenced in https://github.com/GTMEGA/StructureLib/blob/master/src/main/java/com/gtnewhorizon/structurelib/StructureLib.java